pyk/src/tests/integration/cterm/test_simple.py

```diff
@@ -35,6 +38,65 @@
 
 SIMPLIFY_TEST_DATA: Final = (('bytes-return', ('mybytes', '.Map'), (r'b"\x00\x90\xa0\n\xa1\xf1a" ~> .K', '.Map')),)
 
+equals = KLabel(name='#Equals', params=(KSort(name='Bool'), KSort(name='GeneratedTopCell')))
+
+MINIMIZE_CONSTRAINTS_TEST_DATA: Final = (
+    (
+        'no-intersection',
+        (
+            (KApply(label=equals, args=[TRUE, KApply('_<=Int_', args=(intToken(0), KVariable('X')))]),),
```

Comment:
There a few ways to make this read nicer:
* `KLabel` is `Callable` (returns a `KApply`)
* `KLabel.__call__` is overloaded to take `*args: KInner`
* `pyk.prelude.utils.token`
* `pyk.prelude.kint.leInt`

```suggestion
            (equals(TRUE, leInt(token(0), KVariable('X'))),),
```

Reply:
But it seems all constraints follow a certain pattern, so consider extracting a function for that.